docs/source/configuration_file.rst

```diff
@@ -510,6 +511,16 @@ See details in both ``train_validation`` and ``test`` for the contrasts that are
         }
     }
 
+.. note::
+    To get the full advantage of the soft segmentations, the following keys 
+    in the config file must be changed: 
+    (i) ``final_activation: relu`` - to use the normalized ReLU function
+    (ii) ``binarize_prediction: -1`` - so that only soft prediction masks 
```

Comment:
This flag is only used for post-processing and doesn't impact training. The current description matches more the one for `soft_gt`. The description should be more like: "to generate soft predictions, i.e., no binarization is applied during inference." 

However, I don't think I would include it because, as described in the paper, you can have superior performance with SoftSeg even with a final binarization step. The 3 params for SoftSeg are really `soft_gt` (use of soft masks during training), `final_activation` to 'relu', and the regression loss.

Reply:
Thanks for the review, @andreanne-lemay! I have updated the description based on your suggestion! In particular, I removed point (ii) from above.